doc/nrf/ug_zigbee_commissioning.rst

```diff
@@ -0,0 +1,203 @@
+It involes an initiator device and a target device, where the initiator is a member of an existing Zigbee network or it can form a new network.
+Once the network is created, the initiator looks for the target on four channels by sending InterPAN frames.
+When the target is found, the initiator can create a local binding with a remote endpoint or request adding the remote endpoint to a group.
+The target first identifies itself with the initiator and then receives requests from it and responds to them.
+
+Touchlink only supports `Distributed security network`_ and it is provided as a ZCL cluster, which needs to be implemented on both the initiator and the target devices.
+
+.. _zigbee_commissioning_modes_ns:
+
+Network Steering
+================
+
+The Network Steering mode is used for finding open networks that a device can attempt to join or open a network for a specified period of time.
+
+.. figure:: /images/zigbee_commissioning_steering.svg
+   :alt: Zigbee Network Steering procedure
+
+   Zigbee Network Steering procedure
+
+How the node acts in this mode depends on whether the node is in a Zigbee network or not:
+
+* If the node is already a member of a Zigbee network, it opens the network and then closes it after the specified period of time (180 seconds by default).
+  The opening of the network is triggered when any of the Router or Coordinator nodes broadcasts the permit join request.
+* If the node is not a member of a Zigbee network, it scans channels for an open network to associate with.
+  Once it finds a suitable network, it attempts to join it and get the credentials.
+  If the node finds more than one available network, it will attempt to join them individually.
+
+If the node does not find a network to join and it has either the Coordinator or the Router role, it can attempt to form a new network.
+If the node has the End Device role, its commissioning procedure stops here.
+
+The new device must be authenticated and authorized to become part of the network.
+For this purpose, it need credentials, which usually take form of a network key.
+For `Centralized security network`_, the authentication and authorization also involves establishing a unique Trust Center Link Key (TCLK).
+
+Network Steering in the |NCS|
+   In the ZBOSS stack, Network Steering is started using the `ZB_BDB_NETWORK_STEERING`_ bit mask.
+
+   In the |NCS|, Network Steering for a node on a network can be started using the following additional options:
+
+   * When running the :ref:`Zigbee Coordinator sample <zigbee_network_coordinator_sample>`, you can press the associated button.
+   * If you are using the :ref:`lib_zigbee_shell` library in your application, you can use the ``bdb start`` command.
+     This command always starts Network Steering for the Router role, while for the Coordinator role it only works when the device is in the network.
```

Comment:
You can consider adding "and when device is not in the network, `bdb start` starts Network Formation"